### DISCUSSION TITLE
How to filter a no relation field with a select

### DISCUSSION THREAD

**Discussion by Lerma4** [Power-Components/livewire-powergrid#334] [Q&A] [answered]
Hello,
I have tried with the "makeInputSelect" with this code:

            Column::add()
                ->title('STATE')
                ->field('state')
                ->sortable()
                ->searchable()
                ->makeInputSelect(Wedding::select('state')->distinct()->get(), 'state', 'state', ['live-search' => false]),

"Wedding" is the same table where I want search the "state".
This way the display of the select is correct, but when I select one of the possible choices the table does not work.

**Comment by dansysanalyst** [comment, marked as answer]
Hi @Lerma4,

Thank you for your question.

I have opened an Issue https://github.com/Power-Components/livewire-powergrid/issues/338 and will update this discussion once the issue is closed.

Thank you for using PowerGrid!
